### ISSUE TITLE
Duplicate card 

### ISSUE THREAD

**Issue by yoobi** [braintree/braintree-web-drop-in#736]
### General information

* SDK/Library version: Javascript V3 SDK + Java SDK
* Environment: Sandbox
* Browser and OS: Chrome Version 90.0.4430.212 (Official Build) (x86_64) // MacOS. 10.14.6

### Issue description

I'm using drop-in UI for my payment,  I'm displaying the card layout with registered card of the user, I've notice that the user can save multiple times the same card (image1) but it is displayed only once in the the UI (image2).

## Image 1
<img width="760" alt="Screenshot 2021-05-24 at 18 29 57" src="https://user-images.githubusercontent.com/17434735/119378434-94220e00-bcbe-11eb-98a7-f056accfce8d.png">

## Image 2
<img width="768" alt="Screenshot 2021-05-24 at 18 33 22" src="https://user-images.githubusercontent.com/17434735/119378555-abf99200-bcbe-11eb-9da7-d5174c54a1bb.png">

Unfortunately there is a bug because when I delete the card used via the UI, the same card appear again and I have to delete it again probably because the first deletion was on **46w4cqg** and the second is for **kdkv3yb**.

The important issue is that if a user paid 20 times and each time the same card was saved he has to delete it 20 times via the UI which is not acceptable for a user-friendly UI.

![drop-ui](https://user-images.githubusercontent.com/17434735/119379042-41952180-bcbf-11eb-8ec3-ddb8d0f2d306.gif)

Question:
Do you have a solution to avoid inserting the same card over and over again ?
Or delete all the duplicate cards of the user at the same time ?

**Comment by BotanAtomic** [comment]
I have the same issue..

**Comment by crookedneighbor** [comment]
We have a task in our backlog to allow you to specify if you do not want to de-duplicate cards for cases like this. Will update here when the task is complete.

**Comment by yoobi** [comment]
Ok cool, do you know when it'll be push to production ?

**Comment by crookedneighbor** [comment]
No, there are no timelines for a resolution for this issue. We'll update this issue when any new information is available.
